addons/website/static/src/snippets/s_embed_code/options.js

```diff
@@ -0,0 +1,111 @@
+odoo.define('website.s_embed_code_options', function (require) {
+'use strict';
+
+const Dialog = require('web.Dialog');
+const core = require('web.core');
+const options = require('web_editor.snippets.options');
+
+const _t = core._t;
+
+options.registry.EmbedCode = options.Class.extend({
```

Comment:
Allow to Center ? 

If you want to center an embed video from youtube you have no way for now...

Reply:
You can `<center><iframe [...]/></center>`
But having a button would be nicer indeed